src/scripts.js

```diff
@@ -82,41 +95,60 @@ roomFilterBtn.addEventListener("click", function() {
 });
 
 document.addEventListener("click", function(e) {
-  if(e.target && e.target.className === 'filter-book-btn'){
+  if (e.target && e.target.className === 'filter-book-btn') {
     postData(customer.bookByFilterRooms(e.target.parentElement.id))
-    .then(() => {
-      Promise.all([fetchData('customers'), fetchData('bookings'), fetchData('rooms')])
-      .then (data => {
-          customerData = data[0].customers
-          bookingData = data[1].bookings
-          roomData = data[2].rooms
-          bookings = new Booking(bookingData)
-          viewUserDash(customer, bookingData, roomData)
-        });
-    })
+      .then(() => {
+        Promise.all([fetchData('customers'), fetchData('bookings'), fetchData('rooms')])
+          .then(data => {
+            customerData = data[0].customers
+            bookingData = data[1].bookings
+            roomData = data[2].rooms
+            bookings = new Booking(bookingData)
+            viewUserDash(customer, bookingData, roomData)
+          });
+      })
   }
 });
 
-returnToDash.addEventListener("click", function() {
-  hide(viewRooms)
-  show(userDashBoard)
-})
-
-
 
 
 
 
+function instanceOfGuest(data) {
+  customer = new Customer(data)
+  hide(loginPage)
+  show(userDashBoard)
+}
+
+function login() {
+  let username = usernameInput.value;
+  let password = passwordInput.value;
+  let userId = username.split('customer')
+  if (username.startsWith('customer') && (userId[1] > 0 && userId[1] < 51) && password === 'overlook2021') {
+    userId = Number(userId[1])
+    getGuest(userId)
+  } else {
+     'Sorry incorrect infomation try again'
+  }
+}
 
+function getGuest(userId) {
+  fetchSingleApi(userId)
+    .then(data => {
+      instanceOfGuest(data)
+      viewUserDash(customer, bookingData, roomData);
+    })
+    .catch(error => console.log(err))
```

Comment:
Here is where you could do an error handling message for your user to view. Maybe something like `errorMessage.classList.remove('hidden')` that will toggle something in your html to unhide a message letting the user know the data could not be received.